### ISSUE TITLE
Its detected?

### ISSUE THREAD

**Issue by Endermagekkj** [deadly/valorant-agent-yoinker#27]
Its detected rn? like if i use it i will get 100% chance suspension or what

**Comment by Neefs** [comment]
No, there is a chance for a suspension but not a ban. It will be a 2 week abi abuse suspension. To bypass this issue download the source code. Run 
`pip install -r requirements.txt`
`cd src/`
`python main.py`.

Also set appropriate delays. Comment on this your results. I have been doing this for months and it has been working.
